vocab-builder-cli/src/main/java/com/github/tkurz/sesame/vocab/Main.java

```diff
@@ -258,10 +258,10 @@ private static Options getCliOpts() {
                 .create('c'));
 
         o.addOption(OptionBuilder
-                .withLongOpt("stringConstantPrefix")
```

Comment:
Should we allow the CLI to set stringConstantPrefix still or will it always be "_" to ensure the field names are all valid and distinct from the URIs?